src/app/shared/modals/search-case-modal/search-case-modal.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'app-search-case-modal',
+  templateUrl: './search-case-modal.component.html',
+  styleUrls: ['./search-case-modal.component.scss'],
+})
+export class SearchCaseModalComponent {
+  filtersForm = new FormGroup({});
```

Comment:
you're using nameUuidEpidNumberLike in the template, but it's not defined here in the form